juno/assetlist.json

```diff
@@ -672,8 +672,8 @@
       ],
       "base": "cw20:juno1cltgm8v842gu54srmejewghnd6uqa26lzkpa635wzra9m9xuudkqa2gtcz",
       "name": "Fanfury",
-      "display": "fury",
-      "symbol": "FURY",
+      "display": "fury.legacy",
```

Comment:
Don't update the display line. It's only used for determining the exponent--it's never actually displayed. I'd recommend instead changing the Name field.